configure.ac

```diff
@@ -430,6 +430,8 @@ MINGW=`echo $host_os | grep mingw32  2>/dev/null`
 AM_CONDITIONAL(MINGW, test ! -z "$MINGW" )
 if test ! -z "$MINGW"; then
 	WSOCKLIB="-lws2_32"
+	LDFLAGS="-no-undefined $LDFLAGS"
+	AC_DEFINE(HAVE_GETTIMEOFDAY)
```

Comment:
guys, don't waste too much time on autotools, I am working on the one unified cmake build system to rule them all ;-). that being said, Windows support is almost done there, there's just one thing remaining for the automated visual studio builds (and later on mingw as well...) feel free to take a look at the appveyor build logs and see if you can help out. and: big thanks for your contributions so far!